scripts/upgrade.sh

```diff
@@ -7,8 +7,9 @@ usage() {
 	echo "  test         Run tests"
 	echo "  deploy       Run deployments"
 	echo "Options:"
+	echo "  --sepolia    Use sepolia environment for command"
 	echo "  --staging    Use staging environment for command"
```

Comment:
we need to change staging as well to point to the new staging? can wait for another PR though but we need to track it -> lets create an issue

Reply:
https://github.com/kkrt-labs/kakarot-rpc/issues